data/types/device/sensor/devicesensor.go

```diff
@@ -1,4 +1,4 @@
-package devicesensor
+package sensor
```

Comment:
Suggest that this file be renamed to `sensor.go` for consistency.

Reply:
agreed